students/jeonghyun-ahn/westagram/users/views.py

```diff
@@ -0,0 +1,37 @@
+import json, re
+from .models            import User
+from django.db.models   import aggregates
+from django.http        import JsonResponse
+from django.views       import View
+
+class UserView(View):
+    def post(self, request):
+        data        = json.loads(request.body)
+        id_1        = data['email']
+        input2      = data['password']
+        nickname    = data['nickname']
+        phone_check = re.compile('^[0-9]{3}[0-9]{3,4}[0-9]{4}')
+        email_check = re.compile('^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$')
+
+
+        if len(id_1) <= 0 or len(input2) == 0 :
+                return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+        elif not email_check.match(id_1) and not phone_check.match(id_1):
```

Comment:
위의 if 조건이 True가 되어 코드가 동작한다면 return이 되기 때문에 굳이 elif로 조건을 묶으시지 않으셔도 됩니다.
아래 elif들도 정말 elif가 필요한 상황인지 생각해 보시고 수정해 주세요.
```suggestion
        if not email_check.match(id_1) and not phone_check.match(id_1):
```